Documentation/Meeting Notes/Group Meeting Minutes.md

```diff
@@ -56,3 +56,29 @@ Discussing/finalizing tech stack choice
 ###### 12:40-12:55pm:
 Discussing report and assigning parts
 
+
+
+## October 23rd
+
+###### 11:15-11:20 pm: 
+Discussing the weekly plans
+
+###### 11:20-12:06 pm: 
+Creating issues and assigning parts
+
+
+
+
+
+
+## October 27th
+
+###### 1:30-1:40pm: 
+Introducing the progress
+
+###### 1:40-1:50pm:
+Discussing the gitstats strategy
+
+###### 1:50-3:00pm:
+Creating issues, assigning parts and discussing the plans of next week 
```

Comment:
It could be really useful to introduce more detail to these notes. For example, I posted a summary of what we discussed in chat once Alvin got back - it would be really nice if people who miss the meeting could just check these notes and catch up rather than having to make a record in two places. So, in the future aim to have enough detail that someone who missed the meeting could understand the key takeaways by reading these notes.